### PR TITLE
Front end changes

### PR DESCRIPTION
**Pull request by harshahemanth**
Not sure if the Lookup feature works properly

### CODE REVIEW

File: safety-quiz/templates/login.html [hunk 1]
```diff
@@ -1,26 +1,58 @@
 {% extends 'layout.html' %}
 {% block page %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (alexdlukens) [inline safety-quiz/templates/login.html:3]: I believe this import is unnecessary. Could you try removing this and see if it still works? JQuery is supposed to be loaded for all templates in safety-quiz

File: safety-quiz/templates/login.html [hunk 2]
```diff
@@ -1,26 +1,58 @@
...
     </div>
-    <div class="card-body" {% if legacy == True %} hidden {% endif %}>
-        <a class="btn btn-lg btn-success" href="{{ url_for( 'auth.login_google') }}">Click here to login with Google.</a>
+    <div class="card-body" {% if legacy== True %} hidden {% endif %}>
+        <a class="btn btn-lg btn-success" href="{{ url_for( 'auth.login_google') }}">Click here to login with
+            Google.</a>
         <a href="?legacy=true">Legacy login</a>
     </div>
-    <div class="card-body" {% if legacy == False %} hidden {%  endif %}>
+    <div class="card-body" {% if legacy== False %} hidden {% endif %}>
```
Comment (alexdlukens) [inline safety-quiz/templates/login.html:14]: try to reset areas where only whitespace is changed. This (and the above change) didn't change the functionality of the code.

File: safety-quiz/templates/register.html [hunk 3]
```diff
@@ -23,27 +23,30 @@
...
                 {% endfor %}
                 <option value="-1">Other not listed.</option>
             </select>
+            <input class="form-control" type="text" id="custom-major" name="custom-major" hidden>
+
         </div>
         <div class="form-group">
             <label for="college">College</label>
-            <select class="form-control" id="college" name="college" required>
+            <select class="form-control" id="college" onchange="validateCName()" name="college" required>
```
Comment (alexdlukens) [inline safety-quiz/templates/register.html:42]: Minor change, could you expand the function name so it is easier to tell what the function does? "validateCollegeName" or "validateMajorName" is easier to understand than abbreviations

**Review by alexdlukens — COMMENTED** [review]
Looks great overall, I see no issue with the changes. We should test the registration flow throroughly (make new, dummy users) before pushing to master. @pkarnia would be able to provide more advise on how to do this. In the meantime, if you need something to work on message me on slack, I have another webpage that could use some work. Cheers!

File: safety-quiz/templates/login.html (hunk 1 above)
Comment (harshahemanth) [reply safety-quiz/templates/login.html:3]: Yes, It's working without the import
I'll update this

File: safety-quiz/templates/register.html (hunk 3 above)
Comment (harshahemanth) [reply safety-quiz/templates/register.html:42]: Updated names

**Review by AlirpJ — APPROVED** [review]

**Review by pkarnia — APPROVED** [review]

**Pull request merged**